cmd/ak/cmd/triggers/list.go

```diff
@@ -46,14 +46,14 @@ var listCmd = common.StandardCommand(&cobra.Command{
 		ctx, cancel := common.LimitedContext()
 		defer cancel()
 
-		ms, err := mappings().List(ctx, f)
+		ms, err := triggers().List(ctx, f)
 		if err != nil {
 			return err
 		}
 
 		if len(ms) == 0 {
-			var dummy *sdktypes.Mapping
-			return common.FailIfNotFound(cmd, "mappings", dummy)
+			var dummy *sdktypes.Trigger
+			return common.FailIfNotFound(cmd, "triggers", dummy)
```

Comment:
dummy?

Reply:
@efiShtain IIRC I wrote this because I needed a nil-by-definition-but-typed instance to pass to `FailIfNotFound`. I think it's a reasonable name.

Reply:
removed dummy, explicitly set type

Reply:
actually scratch that. removed check entirely - failifnotfound should not be allowed on list results.

Reply:
No, you need this check because all `list commands have a `--fail` flag (i.e. return code = 1 if list is empty). This is the standard way right now to implement it, by reusing the check which was originally written for `get` commands.

Reply:
i'll keep the failifnotfound for now, but let's discuss as part of ENG-451